client/src/style/style.scss

```diff
@@ -2,7 +2,14 @@
 @import "~@coreui/coreui/scss/coreui.scss";
 
 // Temp fix for reactstrap
-@import '~@coreui/coreui/scss/_dropdown-menu-right.scss';
+@import "~@coreui/coreui/scss/_dropdown-menu-right.scss";
 
 // ie fixes
 @import "ie-fix";
+
+.app-header {
+  position: unset !important;
+}
+.app-body {
+  margin-top: 0 !important;
+}
```

Comment:
Is it okay to write styles directly to here or should we add them as a **separate** file and write `@import` directive to here ? I guess it would be better to create a separate file like `header.scss` and import it here.

Reply:
@gulistanboylu, Core UI already have these separate files and import in order to override structure in original repository. We can keep using the same structure as they recommended inside the files (Please check custom.scss and variables.scss).